test/test_rospy/test/unit/test_rospy_core.py

```diff
@@ -209,14 +336,6 @@ def test_is_topic(self):
                 self.fail(f)
             except rospy.core.ParameterInvalid:
                 pass
-            
-    def test_configure_logging(self):
-        # can't test actual functionality
-        try:
-            rospy.core.configure_logging("/")
-            self.fail("configure_logging should not accept a the root namespace as the node_name param")
-        except: pass
-        rospy.core.configure_logging("/node/name")
```

Comment:
I see the test has been moved here https://github.com/ros/ros_comm/pull/1144/files#diff-719224e090245a8163761263deeb3355R91 (slightly different line than the above link. LGTM.